src/WingetCreateCLI/Properties/Resources.resx

```diff
@@ -1471,4 +1471,13 @@ Warning: Using this argument may result in the token being logged. Consider an a
   <data name="MicrosoftEntraIdAuthenticationInfo_Scope_KeywordDescription" xml:space="preserve">
     <value>The scope value for Microsoft Entra Id authentication</value>
   </data>
+  <data name="DownloadProtocolNotSupported_Error" xml:space="preserve">
+    <value>Only HTTPS URL are supported for downloads. Use the "--allow-unsecure-downloads" option to allow HTTP URLs.</value>
+  </data>
+  <data name="DownloadHttpsOnly_Error" xml:space="preserve">
+    <value>Only HTTPS URLs are supported without "--allow-unsecure-downloads".</value>
+  </data>
+  <data name="AllowUnsecureDownloads_HelpText" xml:space="preserve">
+    <value>Allow unsecure downloads (HTTP) for this operation.</value>

```

Comment:
```suggestion
    <value>Allow unsecure downloads (HTTP) for this operation. Default is false.</value>
```

For consistency with other help texts of boolean params

<img width="1333" height="408" alt="image" src="https://github.com/user-attachments/assets/79fca104-c103-4a21-9d1b-cf2be6295e88" />

Reply:
Agree this does fall into a bit of a gray area 😉 That said, I think it's reasonable to assume that if the option isn't explicitly specified, the feature should be considered disabled by default.